runtime/src/tests/storage_integration.rs

```diff
@@ -0,0 +1,42 @@
+use super::initial_test_ext;
+use crate::integration::storage::StorageProviderHelper;
+use crate::Runtime;
+
+use bureaucracy::{Instance2, Worker};
+use srml_support::{StorageLinkedMap, StorageMap};
+
+#[test]
+fn storage_provider_helper_succeeds() {
+    initial_test_ext().execute_with(|| {
+
+		// Error - no workers.
```

Comment:
formatting seems a little off here, but cargo-fmt isn't complaining, tab vs space indentation

Reply:
Added one line break. cargo fmt is still OK.